package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "similar-npm-packages",
+  "version": "1.0.0",
+  "description": "Similar packages suggestions for the NPM ecosystem",
+  "main": "dist/index.js",
+  "scripts": {
+    "lint": "eslint src",
+    "check-types": "tsc --noEmit true",
+    "test": "npm run lint && npm run check-types",
+    "build": "rm -rf ./dist/* && tsc --outDir dist",
+    "prepublishOnly": "npm run build"
+  },
+  "repository": "pastelsky/similar-npm-packages",
```

Comment:
Could we add the full github link here?

Reply:
As far I know all systems (including NPM and bundlephobia) understand the short statements.
But I could insert the full link if you want.

Reply:
Ah, I forget how much of a default github has become now. That's fine then.